src/noshow/dashboard/layout.py

```diff
@@ -9,24 +9,19 @@
 from noshow.dashboard.helper import (
     highlight_row,
     navigate_patients,
+    navigate_uncalled,
     next_preds,
     previous_preds,
     search_number,
     start_calling,
 )
 from noshow.database.models import ApiCallResponse, ApiPatient, ApiSensitiveInfo
 
-STATUS_LIST = [
-    "Niet gebeld",
-    "Wordt gebeld",
-    "Gebeld",
-    "Onbereikbaar",
-]
 RES_LIST = [
     "Herinnerd",
-    "Verzet/Geannuleerd",
-    "Geen",
     "Voicemail ingesproken",
+    "Verzet/Geannuleerd",
+    "Onbereikbaar",
```

Comment:
Maybe we still want a version of "Geen" so you can reset the value in case you haven't called the patient, but accidentally pressed "start met bellen"